turbopy/diagnostics.py

```diff
@@ -314,6 +319,10 @@ def initialize(self):
 
         # Use composition to provide i/o functionality
         self.outputter = utilities[self._input_data["output_type"]](**self._input_data)
+        
+        # set up interval handler
+        if self.interval:
+            self.handler = IntervalHandler(self.interval, self.csv.write_data)
 
     def finalize(self):
         """
```

Comment:
References to CSV files need to be removed. This now uses the more general `outputter` helper object.